aider/resources/model-metadata.json

```diff
@@ -1,16 +1,32 @@
 {
     "deepseek-reasoner": {
```

Comment:
@paul-gauthier Should this have been `deepseek/deepseek-reasoner`? The cost update I applied here didn't "take", as a result the claimed cost for the reasoner model is about 4 times what I was actually charged.